pkg/apis/serving/v1alpha1/revision_validation_test.go

```diff
@@ -638,11 +641,18 @@ func TestRevisionValidation(t *testing.T) {
 		want: nil,
 	}, {
 		name: "empty spec",
-		r:    &Revision{},
+		r: &Revision{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "valid",
```

Comment:
Is this now required for this test to work, or is this just filling in some forgotten boilerplate on the test table?

Reply:
Yes, the K8s function rejects empty names, so the validation before was overly weak.